lance_ray/datasink.py

```
@@ -234,7 +238,7 @@ def __init__(
 
     @property
     def min_rows_per_write(self) -> int:
-        return self.min_rows_per_file
+        return min(self.min_rows_per_file, self.max_rows_per_file)
```

I’m not sure whether this would confuse users. While it technically works, it risks undermining the intended semantics of `min_rows_per_write`. Instead, should we raise an error?
```
raise ValueError("min_rows_per_file cannot exceed max_rows_per_file")
```
 If users want the currently observed behavior, we should recommend explicitly setting both `min_rows_per_file` and `max_rows_per_file` to the desired value， which can be similar to the original ray's design.